src/main/webapp/admin_dashboard_script.js

```diff
@@ -1,30 +1,158 @@
-google.charts.load('current', {'packages':['line']});
-google.charts.setOnLoadCallback(drawChart);
+var map;
+var styledMapType;
+var map_key = keys.MAP_KEY;
+
+load(`https://maps.googleapis.com/maps/api/js?key=${map_key}`); // Add maps API to html
+
+google.charts.load("current", { packages: ["line"] });
 
+google.charts.setOnLoadCallback(drawChart);
 window.addEventListener("resize", drawChart);
+window.addEventListener("load", drawMap);
+
+//window.onload(drawMap());
 
 function drawChart() {
-	var data = new google.visualization.DataTable();
-	data.addColumn('number', 'Days');
-	data.addColumn('number', 'Admins');
-	data.addColumn('number', 'Consecutive Login');
-	data.addColumn('number', 'Tasks');
-	data.addRows([
-		[1,  37.8, 80.8, 41.8],
-		[2,  30.9, 69.5, 32.4],
-		[3,  25.4,   57, 25.7],
-		[4,  11.7, 18.8, 10.5],
-		[5,  11.9, 17.6, 10.4],
-		[6,   8.8, 13.6,  7.7],
-		[7,   7.6, 12.3,  9.6],
-		[8,  12.3, 29.2, 10.6],
-		[9,  16.9, 42.9, 14.8],
-		[10, 12.8, 30.9, 11.6],
-		[11,  5.3,  7.9,  4.7],
-		[12,  6.6,  8.4,  5.2],
-		[13,  4.8,  6.3,  3.6],
-		[14,  4.2,  6.2,  3.4]
-	]);
-	var chart = new google.charts.Line(document.getElementById('chart-div'));
-	chart.draw(data);
+  var data = new google.visualization.DataTable();
+  data.addColumn("number", "Days");
+  data.addColumn("number", "Admins");
+  data.addColumn("number", "Consecutive Login");
+  data.addColumn("number", "Tasks");
+  data.addRows([
+    [1, 37.8, 80.8, 41.8],
+    [2, 30.9, 69.5, 32.4],
+    [3, 25.4, 57, 25.7],
+    [4, 11.7, 18.8, 10.5],
+    [5, 11.9, 17.6, 10.4],
+    [6, 8.8, 13.6, 7.7],
+    [7, 7.6, 12.3, 9.6],
+    [8, 12.3, 29.2, 10.6],
+    [9, 16.9, 42.9, 14.8],
+    [10, 12.8, 30.9, 11.6],
+    [11, 5.3, 7.9, 4.7],
+    [12, 6.6, 8.4, 5.2],
+    [13, 4.8, 6.3, 3.6],
+    [14, 4.2, 6.2, 3.4],
+  ]);
```

Comment:
Personal suggestion: Make the input of the `addRows()` as a variable for now. For example, define `var rowData = [[1, 37.8, 80.8, 41.8], ..., [14, 4.2, 6.2, 3.4]];` and change line 21 to `data.addRows(rowData);`. After making the change, if we want to replace the hard-coded data with real data, we can just set `rowData` to real data and don't need to make any further changes.